### PR TITLE
TON Bridge

### PR DESCRIPTION
**Pull request by vovac12**
https://github.com/sora-xor/sora2-network/issues/1150
https://github.com/sora-xor/sora2-network/issues/1149

### CODE REVIEW

File: pallets/jetton-app/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+[package]
+name = "jetton-app"
+version = "0.1.1"
+authors = ["Snowfork <contact@snowfork.com>"]
```
Comment (Alexey-N-Chernyshov) [inline pallets/jetton-app/Cargo.toml]: 🤔

File: pallets/jetton-app/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+// USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+//! # EVM Fungible App
+//!
+//! An application that implements bridged fungible (ERC-20 and native) assets.
+//!
+//! ## Overview
+//!
+//! ETH balances are stored in the tightly-coupled [`asset`] runtime module. When an account holder
```
Comment (Alexey-N-Chernyshov) [inline pallets/jetton-app/src/lib.rs]: `ETH balances` - comment not relevant.

File: pallets/jetton-app/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+//! this event and relay it to the other chain.
+//!
+//! ## Interface
+//!
+//! ### Dispatchable Calls
+//!
+//! - `burn`: Burn an ERC20 token balance.
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub const TRANSFER_MAX_GAS: u64 = 100_000;
```
Comment (Alexey-N-Chernyshov) [inline pallets/jetton-app/src/lib.rs]: Is it used?

**Review by Alexey-N-Chernyshov — APPROVED** [review]
Please, review comments. Some references to Eth are present.

File: pallets/jetton-app/src/lib.rs (hunk 3 above)
Comment (vovac12) [reply pallets/jetton-app/src/lib.rs]: Removed

**Pull request merged**